spec/javascript/components/components.test.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { Spinner } from 'patternfly-react';
+
+describe('<Spinner/>', () => {
+  it('renders three <Foo /> components', () => {
+    const wrapper = shallow(<Spinner loading />);
+    expect(wrapper.find('.spinner')).toHaveLength(1);
```

Comment:
I would maybe add a snapshot test here to show off how it works:
```
import toJson from 'enzyme-to-json';
...
  expect(toJson(wrapper)).toMatchSnapshot();
...
```

Reply:
Yup, I owe you this from yesterday, as soon as Travis configuration works I'll do it.